packages/code-studio/src/styleguide/Charts.tsx

```diff
@@ -1,30 +1,19 @@
-import React, { PureComponent } from 'react';
+import React, { ReactElement, useState } from 'react';
 import { Chart, ChartModel, MockChartModel } from '@deephaven/chart';
+import { useApi } from '@deephaven/jsapi-bootstrap';
 
-interface ChartsState {
-  model: MockChartModel;
-}
-class Charts extends PureComponent<Record<string, never>, ChartsState> {
-  constructor(props: Record<string, never>) {
-    super(props);
-
-    this.state = {
-      model: new MockChartModel(),
-    };
-  }
-
-  render(): React.ReactElement {
-    const { model } = this.state;
+function Charts(): ReactElement {
+  const dh = useApi();
+  const [model] = useState(new MockChartModel(dh));
```

Comment:
Should be in the habit of using a function to initialize state if it's creating an object or otherwise doing something "expensive".
```suggestion
  const [model] = useState(() => new MockChartModel(dh));
```